README.md

```diff
@@ -2,31 +2,31 @@
 
 # Node.js Nikita
 
-Nikita gather a set of functions usually used during system deployment.
+Nikita gathers a set of functions commonly used during system deployment.
 Documentation is available on the [project website](https://nikita.js.org).
 
 ## Main features 
 
 * Consistent Usage   
   All the functions share the same API, accepting options and a user callback in a flexible manner. Once you learn the core usage, you only learn the options of the actions you wish to execute.  
 * Everything is a file   
-  No agent to install, no database to depends on. Your project is just another Node.js package easily versionned in Git and any SCM, easily integrated with your CI and CD DevOps tools.
+  No agent to install, no database to depend on. Your project is just another Node.js package, easily versioned in Git or any SCM. Easily integrated with your CI and CD DevOps tools.
```

Comment:
I think `easily...` shouldn't be a new sentence, no verb in it

Reply:
Here 'easily' is an adverb' and 'integrated' is the verb (past participle conjugation of integrate). I suggested the change because the sentence sounds more natural but grammatically it is correct either way.

Reply:
I rather keep it how it was, part of the previous sentence